### PR TITLE
Fixed the gear icon appearing again

### LINKED ISSUE #102: Gear icon is back

### PR DESCRIPTION
**Pull request by dapetcu21**
Fixes #102

### CODE REVIEW

**Comment by orbxball** [comment]
So do we need to download it again or you are going to release a new version?

**Comment by dapetcu21** [comment]
We technically need to wait for @rsms to merge this PR. No need to redownload it after he merges. You can use this in the meantime, though:
https://drive.google.com/file/d/0BwvFPSh42FqxdjNhdlB5NTNwdEE/view?usp=sharing

File: website/app/main.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ window.MacMessenger = {
   showSettings: function() {
     this.openGearMenu();
     document.querySelector(
-      'div.uiLayer.uiContextualLayerPositioner#js_1 > div.uiContextualLayer a'
+      'div.uiLayer.uiContextualLayerPositioner#js_2 > div.uiContextualLayer a'
```
Comment (blakeembrey) [inline website/app/main.js:21]: Do you think we can fix the duplicate selectors and improve this regression? I was thinking about making the selector for the cog more unique (not location based though) - something like `[title*="Settings"]`. Then do a lookup based on that elements "aria-owns" like `document.getElementById(cog.getAttribute('aria-owns')).querySelector('a')`.

Comment (cvx) [inline website/app/main.js:21]: @blakeembrey element's title depends on the language of the interface.

Comment (blakeembrey) [inline website/app/main.js:21]: @CvX Haha, of course. Still, I think consolidating the selectors would be good. Avoids keeping two places up to date. 

Edit: Even if it's just a string somewhere with `js_2` that we reuse in each selector.

Comment (blakeembrey) [inline website/app/main.js:21]: Change the `[title*="Settings"]` to the ID then? `#js_1u`. Anyway, the rest should make sense.

Comment (cvx) [inline website/app/main.js:21]: Element ids are dynamic (e.g. for me the id is currently `js_2z`). It's the same with data-reactid. And classes will also most likely change on update.

Comment (blakeembrey) [inline website/app/main.js:21]: @CvX Hmm, they are definitely all auto-generated. It looks like the class `.uiContextualLayerPositioner` has been a consistent hook then. We can do the reverse order. Select this and use the id to find the thing targeting it. I've got no bright ideas short of just asking them to leave class name hooks for external modification. 

Edit: Or we try hooking into the React runtime which is possible.

**Comment by mattbroussard** [comment]
I think @blakeembrey's idea of hooking into React is a good one, but I'm not sure how to do it. [React Dev Tools](https://github.com/facebook/react-devtools) is able to inspect the hierarchy apparently, so maybe we should look into how it does it?
screen shot 2015-04-25 at 4 35 15 pm

`require('React')` appears to get you the React object on Messenger.com

**Comment by dapetcu21** [comment]
This has been already fixed in master, so I I'll close this PR.

**Pull request closed without merging**